### PR TITLE
[meshery]meshery bash script to allow installation of cli alone

### LINKED ISSUE #3200: [Compatibility] Meshery installer: PopOS
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

Was following guide to install meshery from [here](https://docs.meshery.io/installation/quick-start#install-meshery)
I need to be able to install meshery using `
 curl -L https://git.io/meshery | PLATFORM=docker bash - 
 `
but i am getting this 

image

#### Environment

- Host OS: Pop!_OS v20.10 
- Meshery Server Version: unavailable
- Meshery Client Version: v0.5.23
- Platform: Docker 
- Docker version: 20.10.2

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by hexxdump**
**Description**
meshery installation bash script installs cli and also deploys Meshery server.
Provided an option to install cli and bail out, instead of proceeding with Meshery deployment.

This PR fixes #3200 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 801a21ba62b9bda0eb56b5bc45f67e85d7f80387 at: https://6104268fe28ad80dc5665248--meshery-docs-preview.netlify.app

File: meshery [hunk 1]
```diff
@@ -132,8 +142,8 @@ fi
 
 printf "\nInstalling mesheryctl in $install_dir.\n"
 WHOAMI=$(whoami)
-if mv ${PWD}/$temporary_dir_name/mesheryctl $install_dir/mesheryctl ; then
-  echo "mesheryctl installed."
+if mv ${PWD}/$temporary_dir_name/mesheryctl $install_dir/mesheryctl >/dev/null 2>&1; then
+  echo "mesheryctl binary installed."
```
Comment (navendu-pottekkat) [inline meshery:146]:
```suggestion
  echo "mesheryctl installed."
```

File: meshery [hunk 2]
```diff
@@ -162,7 +178,13 @@ else
...
 
 
-printf "Removing installation files and opening Meshery..."
+printf "Removing installation files"
 rm -rf meshery.zip ${PWD}/$temporary_dir_name/
 
-mesheryctl system start --yes -p $PLATFORM
+if [ "$DEPLOY_MESHERY" == false ] ;
+then
+    printf "\nmesheryctl command installation is complete.\n"
```
Comment (navendu-pottekkat) [inline meshery:186]: We say mesheryctl is installed in line 146. Instead we could say "Run mesheryctl system start" to start Meshery. Or something along those lines.

File: meshery [hunk 3]
```diff
@@ -162,7 +178,13 @@ else
...
+    exit 0
+fi
+printf "\nDeploying Meshery...\n"
```
Comment (navendu-pottekkat) [inline meshery:189]:
```suggestion
printf "\nStarting Meshery...\n"
```

**Comment by l5io** [comment]
🚀 Preview for commit fd29e4e73dfa3dd88204d12f6728b9c0ee3045e7 at: https://6104cd3e86f4046914abd0c3--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8b31a4408847c9eada484871915edc6d060a8b3c at: https://6104cf72ac08285e1134dc54--meshery-docs-preview.netlify.app

**Review by navendu-pottekkat — APPROVED** [review]
LGTM 💯

**Pull request merged**